src/redux/actions/transactions.ts

```diff
@@ -140,7 +146,11 @@ export function handlePolkadotTransactionEvents(
   }
 
   if (result.status.isInBlock) {
-    let nonce = result.events[0].event.data[0].toString();
+    let nonce = '0';
```

Comment:
i'm not following 100% why this is done? can you explain why this change?

Reply:
This was to correctly read the nonce from eth and nft transactions but is unrelated to the ERC20 error so I have removed it